lib/src/mock_user.dart

```diff
@@ -74,18 +80,27 @@ class MockUser with EquatableMixin implements User {
   @override
   String? get refreshToken => _refreshToken;
 
+  set idTokenAuthTime(DateTime time) {
+    _idTokenAuthTime = time;
+  }
+
+  set idTokenExp(DateTime time) {
+    _idTokenExp = time;
+  }
+
```

Comment:
I feel like it is more concise to only keep them as constructor parameters. Then you can set the variables as `final` like all the other variables in `MockUser` and usage goes from:
```dart
final user = MockUser();
final customAuthTime = DateTime.parse('2020-01-01');
final customExp = DateTime.parse('2020-01-02');
// Modify MockUser
user.idTokenAuthTime = customAuthTime;
user.idTokenExp = customExp;
```
to
```dart
final customAuthTime = DateTime.parse('2020-01-01');
final customExp = DateTime.parse('2020-01-02');
final user = MockUser(
  idTokenAuthTime: customAuthTime,
  idTokenExp: customExp,
);
```
If we start adding setters for every property, it'll add a lot of boilerplate code.

What do you think? Do you expect people might want to set `idTokenAuthTime` and `idTokenExp` multiple times in their unit tests?

Reply:
I think yes, making it more concise is better. If people want to use another idTokenAuthTime and idTokenExp, they can create another user.